src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2142,9 +2155,16 @@ private void CheckForLatestRender(object obj)
 
         public DynamoViewModel ViewModel { get { return this; } }
 
-        #endregion
-
+        public IBrandingResourceProvider BrandingResourceProvider { get; private set; }
 
+        public ImageSource AboutBoxIcon
+        {
+            get
+            {
+                return BrandingResourceProvider.GetImageSource(ResourceName.AboutBoxLogo);
+            }
+        }
 
+        #endregion
```

Comment:
Consider your very own branding related properties? Or even go as far as having partial class in `DynamoViewModel.Branding.cs`? That's what I would do to avoid adding on to the already lengthy file (and branding stuff form a natural group of *things*). I'm not sure if that makes sense though